spec.html

```diff
@@ -16564,7 +16572,7 @@
               for (const key of Reflect.ownKeys(obj)) {
                 if (typeof key === "symbol") continue;
                 const desc = Reflect.getOwnPropertyDescriptor(obj, key);
-                if (desc && !visited.has(key)) {
+                if (desc) {
```

Comment:
I think this polyfill code might need more tweaking - does it still need `visited` at all?